tests/stdlib/tos.nim

```diff
@@ -145,7 +145,7 @@ else:
   echo getLastModificationTime("a") == tm
 removeFile("a")
 
-when defined(Linux) or defined(osx):
```

Comment:
Would `defined(unix)` be more meaningful? So that BSD and others can also enjoy the party.

Reply:
``defined(posix)`` seems more useful here, yeah.

Reply:
done